chamber/middleware.py

```diff
@@ -0,0 +1,38 @@
+from django.db import transaction
+from django.db import connections
+from django.urls import get_resolver
+
+from chamber.utils.transaction import transaction_signals
+
+
+def get_view_from_request_or_none(request):
+    try:
+        if hasattr(request, 'urlconf'):
+            urlconf = request.urlconf
+            set_urlconf(urlconf)
```

Comment:
isn't the import of this function missing?

Reply:
you are right